packages/core/src/components/widgets/TextareaWidget.tsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { WidgetProps } from "@rjsf/utils";
+
+function TextareaWidget<T = any, F = any>({
+  id,
+  options = {},
+  placeholder,
+  value,
+  required,
+  disabled,
+  readonly,
+  autofocus = false,
+  onChange,
+  onBlur,
+  onFocus,
+}: WidgetProps<T, F>) {
+  const _onChange = ({
+    target: { value },
+  }: React.ChangeEvent<HTMLTextAreaElement>) => {
+    return onChange(value === "" ? options.emptyValue : value);
+  };
+  return (
+    <textarea
+      id={id}
+      className="form-control"
+      value={value ? value : ""}
+      placeholder={placeholder}
+      required={required}
+      disabled={disabled}
+      readOnly={readonly}
+      autoFocus={autofocus}
+      rows={options.rows}
+      onBlur={onBlur && ((event) => onBlur(id, event.target.value))}
+      onFocus={onFocus && ((event) => onFocus(id, event.target.value))}
```

Comment:
Consider refactoring the event handlers out into local fat-arrow functions similar to what you see for `onChange`, typing the event properly